tests/unit/store/settings.spec.js

```diff
@@ -577,18 +596,19 @@ describe("store/settings", () => {
     current_customerids.user_ids = current_list_of_user_ids;
     /* SaveChanges selected */
     store.commit("settings/set_customer_account_ids", current_customerids);
-    let updated_list_of_user_ids =
-      store.getters["settings/user_ids_for_selected_customer_account"];
-    let modified_userids = store.getters["settings/user_details"];
-    expect(modified_userids.uuid).toEqual("6cBaidlJ84Ggc5JA7IYCgv");
-    expect(modified_userids.nickname).toEqual("Updated Account -1");
-    expect(updated_list_of_user_ids.length).toEqual(2);
+    const updated_list_of_user_ids =
+      store.state.settings.customer_account_ids[0].user_ids;
+    const modified_userids =
+      store.state.settings.customer_account_ids[0].user_ids[1];
+    expect(modified_userids.uuid).toStrictEqual("6cBaidlJ84Ggc5JA7IYCgv");
+    expect(modified_userids.nickname).toStrictEqual("Updated Account -1");
+    expect(updated_list_of_user_ids).toHaveLength(2);
   });
-  test("Store has a single customer  with multiple user ids can be assigned to the customer_ids array an 'Edit User' ==>  PopopUpDialog (Delete ID) ==> SettingsForm (SaveChanges) ", () => {
+  test("Given the Vuex has customer details with multiple user id, When the user deletes one of the User ID details, Then validate that the number of user ids is reduced by one in the Vuex", () => {
```

Comment:
same comment as previous test about deleting customers.  All that's being tested here is that committing an array causes the values in that array to be stored in Vuex.  The splicing and other things that mimic "deleting" are happening within the test itself--the test is not checking that any part of the main code is capable of handling a "delete" situation.  Please update the test title to reflect what is being tested here